PWA/.env

```diff
@@ -1,2 +1,2 @@
-VUE_APP_SERVER_URL=https://beta-iddappserver.azurewebsites.net/
```

Comment:
this should be blocked in the .gitignore, maybe you dont have the most recent version yet? Just remove this file from the commit, or you can solve it with any other merge conflicts

Reply:
besides from this, I think it looks good, I have been testing it for a while